apps/ledger-live-mobile/e2e/specs/market.spec.ts

```diff
@@ -0,0 +1,64 @@
+import * as detox from "detox";
+import PortfolioPage from "../models/wallet/portfolioPage";
+import MarketPage from "../models/market/marketPage";
+import { loadConfig } from "../bridge/server";
+import { isAndroid, getElementByText } from "../helpers";
+
+let portfolioPage: PortfolioPage;
+let marketPage: MarketPage;
+
+describe("Market page for user with no device", () => {
+  beforeAll(async () => {
+    await loadConfig("1accountEth", true);
+    portfolioPage = new PortfolioPage();
+    marketPage = new MarketPage();
+  });
+
+  it("should find the researched crypto", async () => {
+    // navigate to the market page
```

Comment:
I would remove these comments as it's fairly clear what is happening in the test script given the function names are clear